suite-common/wallet-utils/src/__fixtures__/ethereumStakingUtils.ts

```diff
@@ -144,7 +144,7 @@ export const getAccountAutocompoundBalanceFixtures = [
     },
 ];
 
-export const getAccountTotalStakingBalanceFixtures = [
+export const getEthAccountTotalStakingBalanceFixtures = [
```

Comment:
since this is only about eth and in the `ethereumStakingUtils` file, it kind of makes sense to have eth in these names

Reply:
Hmm, but the name includes Eth. I made this change to avoid confusion when you import `getAccountTotalStakingBalance` from `wallet-utils`, and the method returns only Ethereum balance.